### PR TITLE
Add explicit mention for code style check

### PR DESCRIPTION
**Pull request by TimvdLippe**
To prevent confusion, let's include it in our CONTRIBUTING.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -82,6 +82,12 @@ For most editors, this should be automatically enforced by [EditorConfig](http:/
 Check if your editor has a built-in plugin or if you need to download one.
 IntelliJ has a built-in plugin, for Eclipse you need to download [this plugin](https://github.com/ncjones/editorconfig-eclipse#readme).
 
+If you want to check if your code complies to the style guide, you can run:
+
+* `./gradlew checkstyleMain` to check the main source code
+* `./gradlew checkstyleTest` to check the test source code
+* `./gradlew build` to check main and test source code, and run the tests
```
Comment (szpak) [inline .github/CONTRIBUTING.md]: `./gradlew test` wouldn't be enough?

Comment (szpak) [inline .github/CONTRIBUTING.md]: `./gradlew check` wouldn't be enough?

Comment (TimvdLippe) [inline .github/CONTRIBUTING.md]: I ran that too and it did the same as `build`. E.g. both check code and run the tests

Comment (szpak) [inline .github/CONTRIBUTING.md]: `check` is (a little bit) cheaper/faster - the artifacts should not be built (at least some of them as acceptance tests depends on some of them, but in general it's good to promote the faster possible way).

It's very similar to `mvn clean install` - a default command in Maven acknowledged by common wisdom. Most of the people in fact would be satisfied with `verify` which doesn't build and install artifacts (sometimes having 100MB+ assuming fat jars) in the local Maven repository.

**Comment by TimvdLippe** [comment]
Merging as the review comment has been resolved.

**Pull request merged**